### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by Dan192000**

### CODE REVIEW

File: values-cs/strings.xml [hunk 1]
```diff
@@ -2,126 +2,126 @@
...
-    <string name="confirm_sign_out_message">You are about to sign out of this game as %1$s.</string>
+    <string name="confirm_sign_out_message">Opravdu se chceš odhlásit ze hry jako %1$s.</string>
     <!-- The format string argument is an email address (i.e. "Currently signed in as: alexjlockwood@gmail.com") -->
-    <string name="currently_signed_in_as">Currently signed in as: %1$s</string>
+    <string name="currently_signed_in_as">Právě přihlášen jako: %1$s</string>
 
     <!-- ABOUT ACTIVITY -->
-    <string name="contact_us">Contact Us</string>
+    <string name="contact_us">Kontaktuj nás</string>
     <string name="developed_and_designed_by">Developed &amp; Designed By</string>
```
Comment (alexjlockwood) [inline values-cs/strings.xml]: Did you forget this string?

File: values-cs/strings.xml [hunk 2]
```diff
@@ -2,126 +2,126 @@
...
-    <string name="special_thanks_to">Special Thanks To</string>
+    <string name="special_thanks_to">Speciální poděkování</string>
     <!-- This is where I will give you credit in the About activity for helping me translate the app. :)
          This string isn't used in the application yet... but it will be when the translations are released! -->
-    <string name="translations_by">Translations By</string>
+    <string name="translations_by">Překlad:</string>
 
     <!-- HOW TO PLAY ACTIVITY -->
```
Comment (alexjlockwood) [inline values-cs/strings.xml]: For these strings, can you also add the <b>...</b> tags as they are used in the English translation? Some words are bold to add emphasis here. :)

File: values-cs/strings.xml [hunk 3]
```diff
@@ -2,126 +2,126 @@
...
+    <string name="how_to_play_144_hint">Nápověda: 2 &#215; 2 &#215; 2 &#215; 2 &#215; 3 &#215; 3 = 144</string>
+    <string name="tap_to_play_random_move">Kliknutím zahraješ náhodný tah!</string>
 
     <!-- SHARING STRINGS -->
     <!-- The title of the activity chooser when you click the "Share" preference in the Settings -->
-    <string name="share_with">Share with</string>
+    <string name="share_with">sdílet</string>
     <!-- The default share message that is used when you click "Share" in the Settings and share with Google+, Facebook, etc. -->
-    <string name="share_app_message">Check out #2048++ on the Play Store!\n\nhttps://play.google.com/store/apps/details?id=com.alexjlockwood.twentyfortyeight</string>
+    <string name="share_app_message">Vyzkoušej aplikaci #2048++ v obchodě google play!\n\nhttps://play.google.com/store/apps/details?id=com.alexjlockwood.twentyfortyeight</string>
```
Comment (alexjlockwood) [inline values-cs/strings.xml]: Should "Google Play" be capitalized?

File: values-cs/strings.xml [hunk 4]
```diff
@@ -2,126 +2,126 @@
...
     <!-- The default share message that is used when you click "Share Screenshot" and share with Google+, Facebook, etc.
          The first and second format string arguments are the game score and game name respectively
          (i.e. "I scored 20188 in a game of Blitz while playing #2048++!)" -->
-    <string name="share_screenshot_message">I scored %1$d in a game of %2$s while playing #2048++!\n\nCheck out the app on the Play Store! https://play.google.com/store/apps/details?id=com.alexjlockwood.twentyfortyeight</string>
+    <string name="share_screenshot_message">Skóroval jsem %1$d v herním módu %2$s ve hře #2048++!\n\nZkus tuto hru také v obchodě google play! https://play.google.com/store/apps/details?id=com.alexjlockwood.twentyfortyeight</string>
```
Comment (alexjlockwood) [inline values-cs/strings.xml]: Should "Google Play" be capitalized?

File: values-cs/strings.xml [hunk 5]
```diff
@@ -2,126 +2,126 @@
...
+    <string name="rate">Ohodnotit</string>
+    <string name="share">Sdílet</string>
+    <string name="show_us_your_support">Ukaž nám svou podporu!</string>
+    <string name="share_with_your_friends">Sdílej 2048++ s přáteli!</string>
+    <string name="google_play_games">Google Play Hry</string>
+    <string name="sign_in">Přihlásit se</string>
+    <string name="sign_out">Odhlásit se</string>
+    <string name="please_sign_in_again">Prosím přihlaš se znovu</string>
+    <string name="sign_in_with_google_play_games">Přihlaš se přes Google Play hry</string>
+    <string name="manage_your_google_play_games_settings">Spravuj nastavení google play her</string>
```
Comment (alexjlockwood) [inline values-cs/strings.xml]: Should "Google Play" be capitalized?

File: values-cs/strings.xml [hunk 6]
```diff
@@ -2,126 +2,126 @@
...
-    <string name="time">Time</string>
-    <string name="sign_in_with_google_dialog_message">Sign in with Google to earn achievements and sync your high scores with the cloud.</string>
-
+    <string name="are_you_sure_warning">Opravdu?</string>
+    <string name="start_new_game_warning">Spuštění nové hry smaže vaší předchozí hru.</string>
+    <string name="start_new_game_warning_current">Spuštění nové hry smaže vaší aktuální hru.</string>
+    <string name="game_over">Konec hry</string>
+    <string name="moves">Pohybů</string>
+    <string name="time">Čas</string>
+    <string name="sign_in_with_google_dialog_message">Přihlaš se přes google k udělování úspěchů a synchronizací vašeho nejvyššího skóre do cloudu.</string>
```
Comment (alexjlockwood) [inline values-cs/strings.xml]: Should "Google" be capitalized?

File: values-cs/strings.xml [hunk 7]
```diff
@@ -2,126 +2,126 @@
...
+    <string name="how_to_play_fibonacci_hint">Nápověda: 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144&#8230;</string>
+    <string name="how_to_play_144_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. Jakýkoliv počet dlaždic se sloučí do jedné pokuď mají stejné čísla.</string>
+    <string name="how_to_play_144_paragraph2">Po každém pohybu se přidá nová dlaždice kdekoliv na ploše. Hra končí pokuď již nelye provést tah. Dokážeš udělat dlaždici s číslem 144?</string>
+    <string name="how_to_play_144_hint">Nápověda: 2 &#215; 2 &#215; 2 &#215; 2 &#215; 3 &#215; 3 = 144</string>
+    <string name="tap_to_play_random_move">Kliknutím zahraješ náhodný tah!</string>
 
     <!-- SHARING STRINGS -->
     <!-- The title of the activity chooser when you click the "Share" preference in the Settings -->
-    <string name="share_with">Share with</string>
+    <string name="share_with">sdílet</string>
```
Comment (alexjlockwood) [inline values-cs/strings.xml]: Should this string be capitalized? It is used as a dialog title in the app (https://github.com/alexjlockwood/android-2048-strings#sharing-strings).

File: values-cs/strings.xml [hunk 8]
```diff
@@ -2,126 +2,126 @@
...
+    <string name="about">O nás</string>
+    <string name="rate">Ohodnotit</string>
+    <string name="share">Sdílet</string>
+    <string name="show_us_your_support">Ukaž nám svou podporu!</string>
+    <string name="share_with_your_friends">Sdílej 2048++ s přáteli!</string>
+    <string name="google_play_games">Google Play Hry</string>
+    <string name="sign_in">Přihlásit se</string>
+    <string name="sign_out">Odhlásit se</string>
+    <string name="please_sign_in_again">Prosím přihlaš se znovu</string>
+    <string name="sign_in_with_google_play_games">Přihlaš se přes Google Play hry</string>
```
Comment (alexjlockwood) [inline values-cs/strings.xml]: On line 39 "hry" is capitalized". Should it be capitalized here as well?

File: values-cs/strings.xml (hunk 1 above)
Comment (Dan192000) [inline values-cs/strings.xml]: No, I would leave English is here.

**Comment by alexjlockwood** [comment]
Thanks a lot for the quick translation! :)

I've added a few comments about some of the strings in your commit... feel free to respond whenever you have the time!

File: values-cs/strings.xml (hunk 4 above)
Comment (Dan192000) [inline values-cs/strings.xml]: Yes, you're right.

File: values-cs/strings.xml (hunk 7 above)
Comment (Dan192000) [inline values-cs/strings.xml]: Yes, sorry

File: values-cs/strings.xml [hunk 9]
```diff
@@ -2,126 +2,126 @@
...
+    <string name="how_to_play_2048_paragraph2">Po každém pohybu se přidá nová dlaždice kdekoliv na ploše. Hra končí pokuď již nelze provést tah. Dokážeš udělat dlaždici s číslem 2048?</string>
+    <string name="how_to_play_243_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 3 dlaždice se sloučí do jedné pokuď mají stejné čísla.</string>
+    <string name="how_to_play_243_paragraph2">Po každém pohybu se přidá nová dlaždice kdekoliv na ploše. Hra končí pokuď již nelze provést tah. Dokážeš udělat dlaždici s číslem 243?</string>
+    <string name="how_to_play_16384_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 2 nebo 4 dlaždice se sloučí do jedné pokuď mají stejné čísla.</string>
+    <string name="how_to_play_16384_paragraph2">Po každém pohybu se přídají 2 nové dlaždice kdekoliv na ploše. Hra končí pokuď již nelze provést tah. Dokážeš udělat dlaždici s číslem 16384?</string>
+    <string name="how_to_play_blitz_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 2 dlaždice se sloučí do jedné pokuď mají stejné čísla.>/string>
```
Comment (alexjlockwood) [inline values-cs/strings.xml]: This line should end with "</string>". Right now it ends with ">/string>" which will result in a syntax error. :)

File: values-cs/strings.xml (hunk 8 above)
Comment (Dan192000) [inline values-cs/strings.xml]: "hry" is better than "Hry"

File: values-cs/strings.xml (hunk 6 above)
Comment (Dan192000) [inline values-cs/strings.xml]: Probably yes

File: values-cs/strings.xml [hunk 10]
```diff
@@ -2,126 +2,126 @@
...
-    <string name="how_to_play_fibonacci_paragraph2">At the end of each move, <b>1 new tile</b> is placed randomly on the grid. The game ends when you can no longer make a move.</string>
-    <string name="how_to_play_fibonacci_hint"><b>Hint:</b> 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144&#8230;</string>
-    <string name="how_to_play_144_paragraph1">Swipe left, right, up, or down to slide the tiles across the grid. When <b>any number of tiles</b> with same number touch, they merge into one.</string>
-    <string name="how_to_play_144_paragraph2">At the end of each move, <b>1 new tile</b> is placed randomly on the grid. The game ends when you can no longer make a move. Can you create the 144 tile?</string>
-    <string name="how_to_play_144_hint"><b>Hint:</b> 2 &#215; 2 &#215; 2 &#215; 2 &#215; 3 &#215; 3 = 144</string>
-    <string name="tap_to_play_random_move">Tap the grid to play a random move!</string>
+    <string name="how_to_play_2048_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 2 dlaždice se sloučí do jedné pokuď mají stejné čísla.</string>
```
Comment (alexjlockwood) [inline values-cs/strings.xml]: For these "How to Play" strings, can you also add the &lt;b&gt;...&lt;/b&gt; tags as they are used in the English translation? Some words are bold to add emphasis here. :)

File: values-cs/strings.xml [hunk 11]
```diff
@@ -2,126 +2,126 @@
...
+        <item>Nové animace</item>
+        <item>Opraveny chyby &amp; vylepšení</item>
     </string-array>
-    <string name="enjoy">Enjoy!</string>
+    <string name="enjoy">Užijte si to!</string>
 
     <!-- HOW TO PLAY HEX STRINGS -->
-    <string name="how_to_play_hex_paragraph1">Swipe left, right, or diagonally to slide the tiles across the grid. When <b>three 1-tiles</b> or any other <b>two tiles</b> with same number touch, they merge into one.</string>
-    <string name="how_to_play_hex_paragraph2">At the end of each move, <b>1 new tile</b> is placed randomly on the grid. The game ends when you can no longer make a move.</string>
+    <string name="how_to_play_hex_paragraph1">Přejeď doleva, doprava nebo diagonálně k posunutí dlaždice po ploše.3 dlaždice s číslem 1 nebo jakékoliv 2 dlaždice se sloučí do jedné pokuď mají stejné čísla.</string>
```
Comment (alexjlockwood) [inline values-cs/strings.xml]: Should there be a space in between ".3" since it is the start of a new sentence?

File: values-cs/strings.xml (hunk 9 above)
Comment (Dan192000) [inline values-cs/strings.xml]: I see it, my fault.

File: values-cs/strings.xml (hunk 10 above)
Comment (Dan192000) [inline values-cs/strings.xml]: I noticed that too late, if you want I'll fix it.

File: values-cs/strings.xml (hunk 11 above)
Comment (Dan192000) [inline values-cs/strings.xml]: Yes :)

File: values-cs/strings.xml (hunk 10 above)
Comment (alexjlockwood) [inline values-cs/strings.xml]: All right, cool! Let me know when the typos are fixed and I'll be able to merge. :)

Comment (Dan192000) [inline values-cs/strings.xml]: Now it should be okay.
Dne 23. 2. 2015 21:48 "Alex Lockwood" <notifications@github.com> napsal(a):

File: values-cs/strings.xml (hunk 7 above)
Comment (alexjlockwood) [inline values-cs/strings.xml]: I think this is the last typo and then everything else looks great! (The string should be capitalized, I think). :)

**Comment by alexjlockwood** [comment]
Sorry for the comment spam, and thanks for the quick fixes!

Before I merge the pull request, I'd like to test the strings in the app myself. I'll send you screenshots of what the strings look like and you'll be able to double check whether there are any last minute changes you think you should make. (It'll also give you a chance to double check for typos one last time too). I'll try to send the screenshots later tonight after work.

After you've verified that the screenshots look OK and that there are no typos, I'll publish your translation in the next release of the application. :)

Thanks again for offering to translate the app! I'm happy to translate it into a new language like Czech. :)

**Comment by Dan192000** [comment]
I like to translate the application, I am waiting for screenshots.
Dne 24. 2. 2015 17:33 "Alex Lockwood" <notifications@github.com> napsal(a):

**Comment by alexjlockwood** [comment]
I just shared the screenshots with you on Google+. If you see any typos or want to change a string for whatever reason, feel free to update the pull request and let me know!

**Comment by Dan192000** [comment]
I fixed two bugs, now it's okay.

Daniel Šíma
Dne 26. 2. 2015 4:13 "Alex Lockwood" <notifications@github.com> napsal(a):

**Comment by alexjlockwood** [comment]
Thanks again for the translation! I'll let you know when it is published!

**Comment by alexjlockwood** [comment]
Hey Daniel, I just published v1.3.2 of 2048++ on the Play Store and it has your new Czech translation. Thanks again for the help! If I ever add new strings to the app (i.e. a new game variant) I'll let you know. :)

**Pull request merged**